src/implementations/twig/pages/page-home/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@ecl/twig-page-home",
+  "author": "European Commission",
+  "license": "EUPL-1.2",
+  "version": "4.6.3",
+  "description": "ECL Example page - Home",
+  "publishConfig": {
```

Comment:
shouldn't this simply be a private package?

Reply:
As I understood, there is no "private" option for npm packages, just "public" or "restricted"
https://docs.npmjs.com/cli/v7/commands/npm-access